### ISSUE TITLE
Suggestion: More biker clothing

### ISSUE THREAD

**Issue by darthvader45** [Flaxbeard/Cyberware#95]
I noticed the mirror shades and biker jacket reminds me of Terminator.  Maybe add biker pants and boots?  It would go well with the jacket and allow us to have that T-800 look with the synth skin. If this can be implemented, thanks.  If not, no biggie.

**Comment by darthvader45** [comment]
Hmm, maybe add the ability to make cybereyes glow like spider eyes in the dark?  Would look creepy.

**Comment by YaibaToKen** [comment]
A red glow, similar to the one seen in the terminator movies on the t-models? That would be awesome

**Comment by darthvader45** [comment]
Yeah, Yaiba.  Exactly.  To go with the clothing.  Especially need biker pants and boots to complete the look of Arnold from T1.

**Comment by Flaxbeard** [comment]
For now, leather pants died a dark grey/light black work well as a substitute - not sure how biker pants would differ

**Comment by LatentDesire** [comment]
Leather chaps?

**Comment by YaibaToKen** [comment]
Iron reinforced leather clothing for added protection? Perhaps the textures could have a bit of a glossier look than vanilla leather

**Comment by darthvader45** [comment]
@Flaxbeard Well, it could have a similar color/look to the biker jacket, which is already equivalent to a leather chestplate.  Plus, the dyed leather still has those out-of-place non-dyed parts.

**Comment by Divineaspect** [comment]
chrome plated

**Comment by YaibaToKen** [comment]
Dynamically generate armoured jackets by using the chestplates present in the game crafted together with the biker jacket *.* Using reflection you should even be able to extend the armour's class to maintain all of it's functionality
